### ISSUE TITLE
Infinite "New Client Connecting"

### ISSUE THREAD

**Issue by wargreen** [jean-emmanuel/open-stage-control#42]
Hi! 
After load a session (maybe created with a biggest screen), get : 
```[Renderer process error]
Uncaught IndexSizeError: Failed to execute 'createRadialGradient' on 'CanvasRenderingContext2D': The r0 provided is less than 0.
Source: src/browser/js/app/widgets/sliders/knob.js @ line 218```
When a new client is connecting, get the same error, and an infinite "New client connecting" on the first client.

**Comment by jean-emmanuel** [comment]
Fixed in c89846b :)

**Comment by wargreen** [comment]
Express fix !
Thanks !
